.github/workflows/close-stalled.yml

```diff
@@ -0,0 +1,33 @@
+name: Close stalled issues and PRs
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    if: github.repository == 'aws/amazon-cloudwatch-agent'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          
+          days-before-pr-stale: 7
+          stale-pr-message: 'This PR was marked stale due to lack of activity.'
+
+          days-before-issue-stale: 90
+          stale-issue-message: 'This issue was marked stale due to lack of activity.'
+
+          days-before-issue-close: 30
+          close-issue-message: Closing this because it has stalled. Feel free to reopen if this issue is still relevant, or to ping the collaborator who labeled it stalled if you have any questions.
+
+          # Do not close PRs
+          days-before-pr-close: -1
+          # close-pr-message: Closing this because it has stalled. Feel free to reopen if this PR is still relevant, or to ping the collaborator who labelled it stalled if you have any questions.
```

Comment:
Honestly, I feel like we should just remove this instead of leaving it in commented.